### ISSUE TITLE
Related label information for fields

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/securedrop.org#865]
There are certain form fields which don't have labels related to them or have broken labels with missing id to associate with:
- [x] In https://securedrop.org/directory/ , the `<select>` dropdown needs to have an `id=aria-topic` for the label tag to be related with it.
- [x] In https://securedrop.org (and other parts of the website), the search field needs to have a label associated with it.
- [ ] ~~#887~~

**Comment by chigby** [comment]
The third item on this list I think is ultimately coming from within Django. The wagtail form builder relies on Django to render checkbox list form fields and these are done without proper grouping tags.

Looks like it might be being [worked on](https://code.djangoproject.com/ticket/32338), though?

**Comment by SaptakS** [comment]
I think we can wait on the django PR to be merged to resolve the third point for now. Depending on how much time it takes we can decide if we want to address it on our side.

**Comment by SaptakS** [comment]
Tracking the third point as a [separate issue](https://github.com/freedomofpress/securedrop.org/issues/887). Closing this.
